src/components/elements/tab.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Text } from 'components/elements/typography'
+
+const StyledTab = styled.li`
+    cursor: pointer;
+    padding: 1.8rem 0;
+    border-bottom: 1px solid var(--color-red-2);
+
+    & > p {
+        color: var(--color-red-2);
+    }
+    &.tab-active {
+        border-bottom: 1px solid var(--color-red);
+
+        & > p {
+            color: var(--color-red);
+        }
+    }
+`
+
+class Tab extends Component {
```

Comment:
Can change this to stateless component and move it to the `sideTab` file, since that is the only place where it's being used.